corehq/apps/app_manager/suite_xml/case_tile_templates/bha_referrals.xml

```diff
@@ -0,0 +1,178 @@
+
+<detail id="m12_case_short">
+    <title>
+        <text>
+            <locale id="cchq.case"/>
+        </text>
+    </title>
+
+   <field>
+    <style horz-align="left" vert-align="center" font-size="small">
+      <grid grid-height="1" grid-width="12" grid-x="0" grid-y="0"/>
+    </style>
+    <header>
+      <text/>
+    </header>
+    <template form="plain">
+      <text>
+        <xpath function="concat('**Profile Information**')"/>
```

Comment:
It looks like other templates use `<template form="{FIELD[format]}">` and `<xpath function="{FIELD[xpath_function]}">`. Should that be the case here or is it important that it's static?

Reply:
Thanks, for context I was passed this xml code and only created the corresponding json file, so this is really helpful. I think you're right that this should use that format.